config/components/webhook/kustomization.yaml

```diff
@@ -4,3 +4,33 @@ resources:
 
 configurations:
 - kustomizeconfig.yaml
+
+patches:
+- patch: |-
```

Comment:
Should the two webhooks removed from `config/components/webhook/manifests.yaml` be re-instated as well?